src/iterative_telemetry/__init__.py

```diff
@@ -146,8 +146,11 @@ def _runtime_info(self):
         """
         Gather information from the environment where DVC runs to fill a report
         """
+        major, minor, patch, *_ = sys.version_info
 
         return {
+            "python_version": f"{major}.{minor}.{patch}",
+            "python_version_minor": f"{major}.{minor}",
```

Comment:
What's the point of having both? The naming is confusing too.

Reply:
We might want to know what python implementation this is in the future, maybe worth making it a dict with misc python info right away?

Reply:
@mike0sv BQ?

Isn't there a way to parse the version properly from `major.minor.patch`? Or maybe store major/minor/patch separately. Having the same thing twice looks really odd. `python_version_minor` sounds like just `minor` to me, not `major.minor`. This is just really odd looking.

Reply:
BQ = big query

Doesnt make sense to store major separately - at least until python 4 is released :)
We can store major+minor and patch separately - joining is easier than splitting

Reply:
Sorry for the delay.

Storing `major.minor` and `patch` separately seems equally ugly. Looking at, for example, homebrew analytics, seems like they store their versions normally without duplicated odd version slices.

Googling around, seems like grouping by split string in bq is easily doable, but maybe I'm missing some complications?